docs/sphinx/_toc.yml.in

```diff
@@ -17,6 +17,9 @@ subtrees:
     - file: src/cli-reference
       entries: 
         - file: src/cli-reference/TensileCreateLibrary                    
+    - file: src/general-reference
+      entries:
+        - file: src/general-reference/environment-variables
```

Comment:
Seems like this belongs in contributor/developer guide

Reply:
I agree, but since these files aren't available yet (they will be added in upcoming PRs) I will keep the environment variables file here as a staging area and will move it when the contributors guide is added.